core/src/main/clojure/xtdb/operator/scan.clj

```diff
@@ -64,7 +66,16 @@
                                  (.getObject 0))
                              (time/sql-temporal->micros expr/*default-tz*))
               :now (-> (expr/current-time)
-                       (time/instant->micros))))
+                       (time/instant->micros))
+              :expr (let [param-types (expr/->param-types args)
+                          projection (expr/->expression-projection-spec "_temporal_expression"
+                                                                        (expr/form->expr arg {:param-types param-types})
+                                                                        {:param-types param-types})]
+                      (util/with-open [dummy-rel (vw/open-args alloc [42])
```

Comment:
rather than dummy-rel, we have (the potentially now misnamed) `vw/empty-args` which doesn't require opening/closing

Reply:
Yes, but then I don't get anything out of the EE, ie. nothing to run.

Reply:
Can potentially use a `NullVector` of size 1 here.